kibana/values.yaml

```diff
@@ -53,6 +53,10 @@ service:
   type: ClusterIP
   port: 5601
 
+# Set this if you are setting server.ssl.enabled in Kibana.
+# This value is a hostname accepted by the SSL certificate provided to kibana.
+kibanaSSLHostname: localhost
```

Comment:
Could you also this to the [configuration](https://github.com/elastic/helm-charts/tree/master/kibana#configuration) section of the readme?

Reply:
+1 will do.